extensions/cornerstone/src/utils/stackSync/toggleStackImageSync.ts

```diff
@@ -1,90 +1,81 @@
-import calculateViewportRegistrations from './calculateViewportRegistrations';
-
-// [ {
-//   synchronizerId: string,
-//   viewports: [ { viewportId: string, renderingEngineId: string, index: number } , ...]
-// ]}
-let STACK_IMAGE_SYNC_GROUPS_INFO = [];
-
-export default function toggleStackImageSync({ toggledState, servicesManager, getEnabledElement }) {
+const disableSync = (syncName, servicesManager) => {
```

Comment:
This code will probably get migrated into the sync service at some point as it is fairly generic now.